tests/Configurator/DockerComposeConfiguratorTest.php

```diff
@@ -0,0 +1,325 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Tests\Configurator;
+
+use Composer\Composer;
+use Composer\IO\IOInterface;
+use Composer\Package\Package;
+use PHPUnit\Framework\TestCase;
+use Symfony\Flex\Configurator\DockerComposeConfigurator;
+use Symfony\Flex\Lock;
+use Symfony\Flex\Options;
+use Symfony\Flex\Recipe;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DockerComposeConfiguratorTest extends TestCase
+{
+    const ORIGINAL_CONTENT = <<<'YAML'
+version: '3.4'
+
+services:
+  app:
+    build:
+      context: .
+      target: symfony_docker_php
```

Comment:
The `docker` part of the name is really redundant... :smile:

Reply:
It's just a fixture file, its content doesn't matter.

Reply:
So, where will the actual file live? In symfony/recipes? We should fix it there then...

Reply:
Or is it the plan that https://github.com/dunglas/symfony-docker will be moved under the symfony org?